app/src/pages/inside/stepPage/modals/linkIssueModal/linkIssueModal.jsx

```diff
@@ -204,55 +245,159 @@ export class LinkIssueModal extends Component {
     };
   };
 
+  componentDidMount() {
+    const { intl, activeProject } = this.props;
+    const {
+      modalState: { testItems },
+    } = this.state;
+    const fetchLogs = () => {
+      this.setState({ loading: true });
+      let testItemLogRequest = [];
+      let requests;
+      if (this.isBulkOperation) {
+        testItems.map((elem) => {
+          return testItemLogRequest.push(
+            fetch(URLS.logItemStackTrace(activeProject, elem.path, ERROR_LOGS_SIZE)),
+          );
+        });
+        requests = testItemLogRequest;
+      } else {
+        testItemLogRequest = fetch(
+          URLS.logItemStackTrace(activeProject, testItems.path, ERROR_LOGS_SIZE),
+        );
+        requests = [testItemLogRequest];
+      }
+
+      Promise.all(requests)
+        .then((responses) => {
+          const [testItemRes] = responses;
+          const testItemLogs = this.isBulkOperation
+            ? responses.map((item) => item.content)
+            : testItemRes.content;
+          const items = [];
+          this.isBulkOperation
+            ? testItems.map((elem, i) => {
+                return items.push({ ...elem, logs: testItemLogs[i] });
+              })
+            : items.push({ ...testItems, logs: testItemLogs });
+          this.setState({
+            modalState: {
+              ...this.state.modalState,
+              testItems: items,
+            },
+            loading: false,
+          });
+        })
+        .catch(() => {
+          this.setState({
+            testItems: [],
```

Comment:
For now `testItems` and `seletedItems` wrapped into `modalState` object. I suggest excluding them and to store just in the state.